tests/Integration/IntegrationTest.php

```diff
@@ -44,7 +45,7 @@ abstract class IntegrationTest extends \Test\TestCase
     protected $feedMapper;
 
     /**
-     * @var FolderMapper
+     * @var FolderMapperV2
```

Comment:
`use OCA\News\Db\FolderMapper;` can be removed since the file was deleted (can't mark the actual line 29 here)